pilot/pkg/model/config.go

```diff
@@ -600,11 +622,14 @@ func ResolveShortnameToFQDN(host string, meta ConfigMeta) string {
 // from the generic config registry
 type istioConfigStore struct {
 	ConfigStore
+
+	// Cache is a cache of jwks_uri, it has some expiration logic on updating the URIs.
+	Cache cache.ExpiringCache
```

Comment:
Shall we call it JwksUriCache?

Reply:
For now it should be JwksUriCache, the reason of using general name is after we cache cert, are we going to going to share the same cache or use different one ? personally I prefer share(need some update on current cache implementation, ex support different expire time for each cached item).

Reply:
I think one cache per object type is cleaner. It also makes implementation simpler. Also, if we cache public key directly, we may not need the intermediate JwksUriCache. "Cache" is too general. Maybe call it "AuthCache"/"JwksCache"/"CertCache"?

Reply:
thanks, renamed to AuthCache, will put shared or separated cache discussion in the design doc.